widgets/bat_openbsd.lua

```diff
@@ -0,0 +1,74 @@
+-- {{{ Grab environment
+local setmetatable = setmetatable
+local tonumber = tonumber
+local string = { format = string.format, gmatch = string.gmatch }
+local io = { popen = io.popen }
+local math = { floor = math.floor }
+local helpers = require("vicious.helpers")
+local string = {
+    gmatch = string.gmatch,
+    match = string.match,
+    format = string.format
+
+}
+-- }}}
+local bat_openbsd = {}
+
+local function worker(format, warg)
+    local battery = warg or "bat0"
+    local filter = string.format("hw.sensors.acpi%s", battery)
+    local cmd = string.format("sysctl -a | grep '^%s'", filter)
+    local proc = io.popen(cmd)
+
+    local bat_info = {}
+    for line in proc:lines("*l") do
+        for key, value in string.gmatch(line, "(%S+)=(%S+)") do
+	    key = key:gsub(filter .. ".", "")
+	    bat_info[key] = value
+        end
+    end
+    proc:close()
+
+    -- current state
+    local states = {[0] = "↯", -- high
+                    [1] = "-", -- low
+                    [2] = "!", -- critical
+                    [3] = "+", -- charging
+                    [4] = "n/a", -- critical
+                    [255] = "⌁"} -- unknown
```

Comment:
I think `N/A` (conventionally this repo uses capitalized version) is more suitable for *unknown* state.

Reply:
Done in 7d62dd3. I also reworded the comments.